sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -182,15 +182,6 @@ static const struct snd_soc_acpi_adr_device rt714_3_adr[] = {
 	}
 };
 
-static const struct snd_soc_acpi_link_adr tgl_i2s_rt1308[] = {
```

Comment:
@plbossart duplicate sign-off in commit message and did you mean RT711 in SDW mode and RT1308 in I2S mode?

Reply:
yes, thanks. will fix.